src/plugins/blocksmith/components/imgur-post-block-preview/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Message from '@gdbots/pbj/Message';
+import Blockquote from './Blockquote';
+
+class ImgurPostBlockPreview extends Component {
+  constructor(props) {
+    super(props);
+    this.createEmbed = this.createEmbed.bind(this);
+    this.loadScript = this.loadScript.bind(this);
+  }
+
+  loadScript() {
+    const IMGUR_EMBED_SCRIPT_ID = 'imgur-embed-script';
```

Comment:
@jalopez13 usually we put these kinds of things in between the imports and the class declaration

Reply:
@efjacobson 

Moved in-between imports and class declaration.